python/tk_multi_launchapp/base_launcher.py

```diff
@@ -482,6 +481,31 @@ def launch_from_path_and_context(self, path, context, version=None):
         """
         raise NotImplementedError
 
+    def _sort_func(self, version_string):
+        """
+        Returns a version object that can be used to sort arbitrary version numbers.
+
+        For environment running Python 3.11 or prior it uses the LooseVersion class
+        from distutils.version. For Python 3.12 and later, it uses the
+        version.parse function from the packaging module. If the packaging module
+        is not available, it falls back to the get_clean_version_string function.
+        """
+        if sys.version_info[0:2] < (3, 12):
+            from distutils.version import LooseVersion
+
+            return LooseVersion(version_string)
+
+        try:
+            from packaging import version
+
+            return version.parse(version_string)
+        except ImportError:
+            self._tk_app.log_debug(
+                "The packaging module is not available. "
+                "Falling back to get_clean_version_string for version sorting."
+            )
```

Comment:
This is going to generate a lot of logs given that it's called in a set of data.

Instead of that, could you make a code that initialize the import first (and log if the module is not present) just like how it's done in tk-core?